icubam/db/store.py

```diff
@@ -659,6 +661,7 @@ def get_visible_bed_counts_for_user(self,
     """
     session = self._session()
     user = self._get_user(session, user_id)
+    # TODO: handle user=None here
     if force or user.is_admin:
```

Comment:
We need to handle the case of `user=None` here. I'm not sure if it should be an error or handled the same as admin? (_get_user can return None)

Reply:
It should be an error I think. A missing user should not behave like an admin.

Reply:
Thanks @sgirgin I opened https://github.com/icubam/icubam/issues/180 to address this in a follow up PR.

Reply:
Great, thank you!